ideascube/mediacenter/migrations/0013_escape_summary.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+from django.utils.html import escape
+
+
+def escape_summary(app, schema_editor):
+    Document = app.get_model('mediacenter', 'Document')
+    db_alias = schema_editor.connection.alias
+    for document in Document.objects.using(db_alias).all():
+        document.summary = "<p>{}</p>".format(escape(document.summary))
```

Comment:
> If we don't wrap already existing summaries in a <p>...</p>, we will have sometime a summary wrapped (for new content) and sometime not (for old content). This is a inconsistency in the way we structure our layout.

I think that's fine.

When users will see the problem, they can just go and edit the content, save it, and everything will display correctly.